README.md

```diff
@@ -39,26 +39,22 @@ The Looker Embed SDK uses a fluent interface pattern. The construction of the em
 
 ### Building
 
-First initialize the SDK with address of your Looker server and, optionally, the endpoint on your server that will perform authentication.  (Note: Port must be included if it is required to reach the Looker server from browser clients, e.g. looker.example.com:443)
-These are used by all the embedded content.
+First initialize the SDK with address of your Looker server and the endpoint on your server that will perform authentication. (Note: Port must be included if it is required to reach the Looker server from browser clients, e.g. looker.example.com:443, but the protocol (http/https) should *not* be included.
 
 ```javascript
 LookerEmbedSDK.init('looker.example.com', '/auth')
 ```
 
-Then the embedded content is built using a series of steps to define its parameters. Some if these parameters are optional, and some are mandatory.
+In this example, `/auth` is a backend service that you must implement as described in the [Auth](#the-auth-endpoint) section.
 
-The process starts with creating the builder with an `id`, or a `url` created by the processed described [here](https://docs.looker.com/r/sdk/sso-embed):
+After the SDK is initialized, begin by creating the builder with an `id`. For example, to create a dashboard embed builder:
 
 ```javascript
 LookerEmbedSDK.createDashboardWithId(id)
 ```
 
-or
-
-```javascript
-LookerEmbedSDK.createDashboardWithUrl(url)
-```
+The `createDashboardWithId` function will call your backend `/auth` endpoint and expect a signed embed URL in response. Subsequent embeds can be generated using `createDashboardWithUrl` which accepts a partial URL matching [this form](https://docs.looker.com/reference/embedding/sso-embed#building_the_embed_url), for example: `/embed/dashboards/`. The URL create functions will not call your backend `/auth` service. If you are embedding
```

Comment:
> he `createDashboardWithId` function will call your backend `/auth` endpoint

Apologies if I'm just misunderstanding, but I thought it was the `.build()` method that calls the `/auth` endpoint